src/_cffi_src/openssl/rand.py

```diff
@@ -9,7 +9,6 @@
 """
 
 TYPES = """
-static const long Cryptography_HAS_EGD;
```

Comment:
Should we leave this constant but set to `0`? Then there's no plausible backwards compat concern :-)